mapzap/models/product/product_mapzap.sql

```diff
@@ -0,0 +1,100 @@
+with project_bbl_geometries as (
+    select *
+    from
+        {{ ref('int_zap_project_bbl_geometries') }}
+),
+
+project_geometries_from_map as (
+    select *
+    from
+        {{ ref('int_zap_project_map_geometries') }}
+),
+
+project_details as (
+    select *
+    from
+        {{ ref('stg_dcp__zap_projects') }}
+),
+
+project_bbl_geometries_aggregated as (
+    select
+        project_id,
+        pluto_version,
+        SUM(ST_AREA(bbl_geometry_wkt)) as bbl_areas_sum,
+        ARRAY_AGG(project_bbl) as project_bbls_array,
+        ST_UNION_AGG(bbl_geometry_wkt) as project_geometry_wkt
+    from project_bbl_geometries
+    group by
+        project_id,
+        pluto_version
+),
+
+project_geometries_from_bbls as (
+    select
+        project_id,
+        pluto_version,
+        bbl_areas_sum,
+        NULLIF(ARRAY_TO_STRING(project_bbls_array, '|'), '') as project_bbls,
+        ST_ASTEXT(project_geometry_wkt) as bbls_wkt
+    from project_bbl_geometries_aggregated
+),
+
+project_geometries as (
+    select
+        project_geometries_from_bbls.*,
```

Comment:
Personal note - I really dislike `select *` statements. One in complex/long queries it can become a chore to come in and see what columns intermediate ctes actually have, and two it can make debugging hard if a source dataset has a missing column or something else unexpected (or potentially new and unwanted columns can make it through our pipeline). The first point stands regardless, but I imagine dbt helps with the second point?

Reply:
Edited to tone down my strong language!